### PR TITLE
TMA-920: self contained goodfile spec

### PR DESCRIPTION
**Pull request by kubamahnert**

### CODE REVIEW

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/80/ : SUCCESS in 10s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/893/ : SUCCESS in 37s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1161/ : SUCCESS in 40s

File: spec/integration/command_projects_spec.rb [hunk 1]
```diff
@@ -29,4 +29,19 @@
     expect(@project.blueprint.datasets.count).to eq(4)
     expect(@project.blueprint.datasets(:all, :include_date_dimensions => true).count).to eq(5)
   end
+
+  describe '#get_spec_and_project_id' do
+    before do
+      File.write '.gooddata', { model: 'model.rb', project_id:  @project.pid}.to_json
```
Comment (yenkins) [inline spec/integration/command_projects_spec.rb]: Style/ExtraSpacing: Unnecessary spacing detected.

Comment (yenkins) [inline spec/integration/command_projects_spec.rb]: Style/SpaceInsideHashLiteralBraces: Space inside } missing.

**Comment by gdgate** [comment]
Build failed (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/90/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/907/ : FAILURE in 43s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1175/ : SUCCESS in 42s

**Comment by gdgate** [comment]
Build succeeded (check pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-opensource-repository-check/91/ : SUCCESS in 7s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/908/ : SUCCESS in 40s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1176/ : SUCCESS in 52s

**Comment by panjan** [comment]
ok to merge

**Comment by gdgate** [comment]
Build failed (gate pipeline).

- https://checklist.intgdc.com/job/compliance/job/gooddata-ruby-merge-compliance-check/40/ : SUCCESS in 13s
- https://ci.intgdc.com/job/gooddata-ruby-pronto-code-review-zuul/946/ : SUCCESS in 49s
- https://ci.intgdc.com/job/gooddata-ruby-unit-tests-zuul/1215/ : SUCCESS in 47s
- https://ci.intgdc.com/job/gooddata-ruby-integration-tests-zuul/1381/ : FAILURE in 1h 16m 48s
- https://ci.intgdc.com/job/gooddata-ruby-lcm-integration-tests-zuul/1146/ : FAILURE in 33s

**Pull request merged**